pkgs/by-name/fe/fermi2/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  config,
+  lib,
+  fetchFromGitHub,
+  zlib,
+  stdenv,
+}:
+stdenv.mkDerivation {
```

Comment:
Please override to use gccStdenv on Darwin. (Or patch it to use clang)
```
Running phase: buildPhase
cc -c -g -Wall -O2    rle.c -o rle.o
build flags: SHELL=/nix/store/fhngvmgagwwxhak8nzrx5vqbg4g4qiac-bash-5.2p37/bin/bash
gcc -c -g -Wall -O2 -Wno-unused-function    kthread.c -o kthread.o
/nix/store/fhngvmgagwwxhak8nzrx5vqbg4g4qiac-bash-5.2p37/bin/bash: line 1: gcc: command not found
```

Reply:
I have lost the edits that fixed that. `makeFlags` have been updated. Sorry :/